src/Tizen.NUI/src/internal/EXaml/Action/AddEventAction.cs

```diff
@@ -64,7 +64,7 @@ public void Init()
 
         public void OnActive()
         {
-            if (null != childOp)
+            if (null != childOp && null != (childOp.ValueList[0] as Instance) && null != (childOp.ValueList[1] as Instance))
```

Comment:
You need to check childOp.ValudList.Count, it must be greater than 3
```suggestion
            if (null != childOp && childOp.ValueList.Count > 3 && (childOp.ValueList[0] is Instance instance) && (childOp.ValueList[1] is Instance element))
            {
                int instanceIndex = instance.Index;
                int elementIndex = element.Index;
                int propertyIndex = (int)childOp.ValueList[2];
                int value = (int)childOp.ValueList[3];
                globalDataList.Operations.Add(new AddEvent(globalDataList, instanceIndex, elementIndex, propertyIndex, value));
            }
```

Reply:
> You need to check childOp.ValudList.Count, it must be greater than 3

Done